src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -68,6 +70,8 @@ public AzureDevOpsClient(string gitExecutable, string accessToken, ILogger logge
             };
         }
 
+        public bool AllowRetries { get => _allowRetries; set => _allowRetries = value; }
```

Comment:
Why give this a backing field? Just make it an auto-property.

Reply:
What does that look like?

Reply:
Nvm, found it in VS auto-suggest.